doc/nrf/config_and_build/bootloaders/bootloader_downgrade_protection.rst

```diff
@@ -29,10 +29,10 @@ Software-based downgrade protection
 The |NCS| supports MCUboot's software-based downgrade prevention for application images, using semantic versioning.
 This feature offers protection against any outdated firmware that is uploaded to a device.
 
-To enable this feature, set the MCUboot-specific configuration options ``CONFIG_MCUBOOT_DOWNGRADE_PREVENTION`` and ``CONFIG_BOOT_UPGRADE_ONLY`` for the MCUboot image.
+To enable this feature, set the configuration option :kconfig:option:`CONFIG_MCUBOOT_DOWNGRADE_PREVENTION` for the MCUboot image and :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_OVERWRITE_ONLY` for sysbuild.
 
 .. caution::
-   Enabling ``CONFIG_BOOT_UPGRADE_ONLY`` prevents the fallback recovery of application images.
+   Enabling :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_OVERWRITE_ONLY` prevents the fallback recovery of application images.
```

Comment:
It seems like for nrf53 this is only sensible mode for mcuboot, as default will only apply for app core and not net core 
 and result in version mismatch - application could roll back to previous working version, but netcore will be ignored (am I right?) maybe we should also add note here about this here

Reply:
That's correct. However sometimes you want to have this enabled but in NCS context this is the only use-case where this is required.